docs/cheatsheet_syntax.md

```diff
@@ -32,8 +32,22 @@ Lines starting with:
 
 All the other non-empty lines are considered as executable commands.
 
-Tip: if you are editing cheatsheets in Visual Studio Code, you could enable syntax highlighting
-by installing [this extension](https://marketplace.visualstudio.com/items?itemName=yanivmo.navi-cheatsheet-language).
+> [!TIP]
+> If you are editing cheatsheets in Visual Studio Code, you could enable syntax highlighting
+> by installing [this extension](https://marketplace.visualstudio.com/items?itemName=yanivmo.navi-cheatsheet-language).
+
+If you use `.cheat.md` as an extension of cheat files, you can place commands in code blocks fenced with triple backticks (```` ``` ````).
```

Comment:
This is not restricted to `.cheat.md` extension... It works for `.cheat` files too

Reply:
Thanks!